### ISSUE TITLE
Separation of profit and loss calculation

### ISSUE THREAD

**Issue by FrederikBolding** [rotki/rotki#3451]
## Abstract

Currently the Profit and Loss report looks to be subtracting the loss from the profit and giving a total number (at least as far as I can tell). It would be beneficial to have a setting or a button to switch between views to be able to view both of these numbers side by side instead of their combined sum.

## Motivation

Certain countries (Denmark for instance, possibly others) require you to separately declare profits and losses for cryptocurrency tax purposes.

**Comment by LefterisJP** [comment]
Profit and loss, PnL is quite universal and is cumulative over multiple events. Each even adds to the PnL if it's profitable and subtracts if it's a loss. This is a very universally accepted term in accounting and have never heard of anyone having a problem with this.

Would require some work and complicate the code a bit to do what you are asking. And nobody has ever asked us before so not sure if we will do it soon. Prioritizing is hard with ~400 open requests :raccoon: 

What you can do to get this for your purposes is write a custom spreadsheet rule at the all events exports. Get all negative values of `net_profit_or_loss` column as loss and all the positive values of `net_profit_or_loss`as profit. And sum them up with a spreadsheet rule.

**Comment by FrederikBolding** [comment]
Yeah, it seems a bit weird to me too, if interested you can read more here: https://skat.dk/skat.aspx?oid=2291754 

I would happily be told I have understood it wrong 😄 

If it doesn't get implemented I'll be sure to use your workaround 👍

**Comment by LefterisJP** [comment]
> If it doesn't get implemented I'll be sure to use your workaround

We more than happily also accept PRs :smiley:

and can help you through any questions. Especially when you are in our discord: https://discord.com/invite/aGCxHG7

**Comment by FrederikBolding** [comment]
> > If it doesn't get implemented I'll be sure to use your workaround
> 
> We more than happily also accept PRs 😃
> 
> and can help you through any questions. Especially when you are in our discord: https://discord.com/invite/aGCxHG7

Yeah, that might be a possibility. Need to be more familiar with the codebase though 😄

**Comment by LefterisJP** [comment]
I will close this as not planned since this is not standard in any accounting format I have seen and no more user demand apart from a single person so far. Sorry Frederik, just need to prioritize. Can reopen again later with PRs that are fully tested and working.
